Sources/PackageLoading/Platform.swift

```diff
@@ -20,23 +20,21 @@ private func isAndroid() -> Bool {
         (try? localFileSystem.isFile(AbsolutePath(validating: "/system/bin/toybox"))) ?? false
 }
 
-public enum Platform: Equatable {
+public enum Platform: Equatable, Sendable {
     case android
     case darwin
     case linux(LinuxFlavor)
     case windows
 
     /// Recognized flavors of linux.
-    public enum LinuxFlavor: Equatable {
+    public enum LinuxFlavor: Equatable, Sendable {
         case debian
         case fedora
     }
 }
 
 extension Platform {
-    // This is not just a computed property because the ToolchainRegistryTests
-    // change the value.
```

Comment:
I assume this comment wasn't true 😅? If that's the case though, can it just be a computed property?

Reply:
It was, `ToolchainRegistryTests` is in SourceKit-LSP: https://github.com/apple/sourcekit-lsp/pull/1276#discussion_r1597909530

I prefer it to be stored so we don’t re-run `uname` every time the variable is accessed.